machAeroTutorials/aero_icem.rst

```diff
@@ -75,6 +77,7 @@ To adjust your view of the geometry in ICEM the following functions are possible
     - Drag mouse left/right: Rotate view in viewing plane
```

Comment:
You can also drag the mouse while right clicking to do planar rotation of the viewing plane